pkgs/development/libraries/qt-5/5.5/qtbase/setup-hook.sh

```diff
@@ -91,7 +91,7 @@ _qtSetQmakePath() {
 
 if [ -z "$NIX_QT5_TMP" ]; then
     if [ -z "$NIX_QT_SUBMODULE" ]; then
-        NIX_QT5_TMP=$(mktemp -d)
+        NIX_QT5_TMP=$(pwd)/__nix_qt5__
```

Comment:
Will check/fix about nix-shell.

It will make, because /tmp path was derandomized year ago. I have dedicated /var/buildroot filesystem, and paths inside sandbox still looks like ``/tmp/${pkg}-drv-0/``

btw can we make single setup-hook.sh for all qt5.x? (and may be reduce amount of qtx.x building by default)